metric_system/functions/metric.py

```diff
@@ -54,7 +54,7 @@ def parse_timestamp(cls, str_timestamp: str) -> datetime:
         return datetime.strptime(str_timestamp, "%Y-%m-%d %H:%M:%S.%f")
 
     @classmethod
-    def create_from(cls, metric: Metric):
+    def create_from_metric(cls, metric: Metric):
```

Comment:
I noticed that you changed the method name, which looks like a good change. However you did not change where it is used, so the tests error out. In VSCode if you rightclick on it and do "rename symbol" it should rename all uses. In any case, please run the tests with the updated code and make sure it works